File: lista_01/04/CalculoPeso.java
import java.util.Scanner;

public class CalculoPeso {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int totalPessoas = 7;
        int somaIdades = 0;
        int idade, pessoasMais90 = 0;
        double peso;

        for (int i = 1; i <= totalPessoas; i++) {
            System.out.printf("Digite a idade da pessoa %d: ", i);
            idade = scanner.nextInt();
            System.out.printf("Digite o peso da pessoa %d (em kg): ", i);
            peso = scanner.nextDouble();

            // Soma das idades para cálculo da média
            somaIdades += idade;

            // Contador de pessoas com mais de 90 quilos
            if (peso > 90) {
                pessoasMais90++;
            }
        }

        // Calcula a média das idades
        double mediaIdades = somaIdades / (double) totalPessoas;

        // Exibe os resultados
        System.out.println("Quantidade de pessoas com mais de 90 kg: " + pessoasMais90);
        System.out.printf("Média das idades das sete pessoas: %.2f anos\n", mediaIdades);

        scanner.close();
    }
}

File: lista_01/04/CalculoPeso.py
def coletar_dados_pessoais():
    total_pessoas = 7
    soma_idades = 0
    pessoas_mais_90kg = 0

    for i in range(1, total_pessoas + 1):
        print(f"Digite as informações para a pessoa {i}:")
        try:
            idade = int(input("Idade: "))
            peso = float(input("Peso (em kg): "))
        except ValueError:
            print("Por favor, insira valores numéricos válidos.")
            continue  # Pula para a próxima iteração do loop

        soma_idades += idade

        if peso > 90:
            pessoas_mais_90kg += 1

    media_idades = soma_idades / total_pessoas

    print(f"Quantidade de pessoas com mais de 90 kg: {pessoas_mais_90kg}")
    print(f"Média das idades das sete pessoas: {media_idades:.2f} anos")

if __name__ == "__main__":
    coletar_dados_pessoais()
